resources/utils.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+import dateutil.rrule
+
+def rename_category_for_flattening(category, category_parent=""):
+    """
+    Tidy name of passed category.
```

Comment:
just to make it extra clear maybe add to "by removing extraneous characters such as '_' and '-'"